### PR TITLE
Adapt to dashboard repo movement

### PR DESCRIPTION
**Pull request by sleshchenko**
### What does this PR do?
Adapt to dashboard repo movement

### What issues does this PR fix or reference?
it's part for #19617
Is done in a scope of https://github.com/eclipse/che/issues/19614

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: tests/.infra/crw-ci/dashboard-next/pr-check/Jenkinsfile [hunk 1]
```diff
@@ -181,14 +181,14 @@ pipeline {
         stage("Build and push 'che-dashboard' image"){
             steps {
                 sh """
-                    echo "Download the 'che-dashboard-next' project and checkout to branch"
```
Comment (dmytro-ndp) [inline tests/.infra/crw-ci/dashboard-next/pr-check/Jenkinsfile:184]: "che-dashboard" is common name now, isn't it?

Comment (sleshchenko) [reply tests/.infra/crw-ci/dashboard-next/pr-check/Jenkinsfile:184]: Good catch! Find/replace mistake.

Removed dashboard next notion here and in other places https://github.com/eclipse/che/pull/19618/commits/cf4fd2f896a1a0413ba802e67e7e363e79f58fdf

**Review by dmytro-ndp — APPROVED** [review]
Looks good to me.

**Comment by sleshchenko** [comment]
merging.
@mkuznyetsov if you have any suggestions, please let me know and I'll fix them

**Pull request merged**